Archive/PiBase/Theorems.lean

```diff
@@ -0,0 +1,48 @@
+import Mathlib.Topology.Basic
+import Mathlib.Data.Set.Basic
+import Mathlib.Order.Filter.Basic
+import Archive.PiBase.Properties
+open Topology Set Filter
+
+universe u
+variable (X : Type u) [TopologicalSpace X]
```

Comment:
Some type classes in topology come with their own ‘TopologicalSpace’ structure, which caused type class issues for me when declaring this globally, so I ended up writing this for every separate theorem if needed